invoicer_project/clients/views.py

```diff
@@ -33,17 +32,15 @@ def create(self, request):
     def list(self, request):
         user = get_user_from_jwt(request.headers)
         clients = Client.objects.filter(user_id=user['user_id'])
-        clients_json = ClientSerializer(clients, many=True)
         if len(clients) == 0:
             return JsonResponse(
-                {"first_name": user['first_name'], "last_name": user['last_name'], 'message': 'Not found'}, status=200)
+                {"first_name": user['first_name'], "last_name": user['last_name'], 'message': 'Not found'}, status=404)
+        clients_json = ClientSerializer(clients, many=True)
         data = {"first_name": user['first_name'], "last_name": user['last_name'], 'content': clients_json.data}
         return JsonResponse(data, status=200, safe=False)
-        # can be problems with JS(safe)
 
     def destroy(self, request, client_id):
-        user = get_user_from_jwt(request.headers)
-        client = get_object_or_404(Client, id=client_id, user_id=user['user_id'])
+        client = get_object_or_404(Client, id=client_id)
         client.delete()
         return JsonResponse({'message': 'Deleted!'}, status=204)
```

Comment:
I don't think that this message is valuable in response

Reply:
done